.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,24 @@
-<!--
-Please read and follow these instructions before creating and submitting a pull request:
+<!-- Thanks for contributing! -->
 
-- If you're fixing a bug, ensure you add unit tests to prove that it works.
-- Before adding a feature, it is best to create an issue explaining it first. It would save you some effort in case we don't consider it should be included in node-fetch.
-- If you are reporting a bug, adding failing units tests can be a good idea.
--->
+## Purpose
+...
 
-**What is the purpose of this pull request?**
+## Changes
+...
```

Comment:
Same as for the `Purpose` section.

Reply:
You really seems to like it to be short? :P 
Like if it isn't shorter already then compare to what it initially [was](https://raw.githubusercontent.com/node-fetch/node-fetch/1bd848b6329a68981ab9d9b98eb552f3540ae4a7/.github/PULL_REQUEST_TEMPLATE.md)

i still prefer ... as a indication to "write text here"  but fine, i will cave in and change all your suggestions

Reply:
Yes, it's always better to convey more information in less words.

Reply:
Maybe can remove `- [ ] I updated ./docs/v3-UPGRADE-GUIDE`
it's pretty much done already...

or change it to `- [ ] I updated ./docs/v4-UPGRADE-GUIDE` :P perhaps too early?